### ISSUE TITLE
DST Staging Review (Consider): Reorganize accessibility guidance to more forcefully steer teams away from `role` attributes

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/vets-design-system-documentation#3675]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Design System Staging Information
**Component:** Alert Sign-in
**Staging Review ticket:** [Design System Staging Review: Alert Sign-in](https://github.com/department-of-veterans-affairs/va.gov-team/issues/99891)

## Findings details
The details of this ticket are for consideration only. No Staging Review issue was found.
**High-priority:** No
**Collab Cycle Reviewer:** @briandeconinck (Accessibility)

### Description

As currently written, the accessibility guidance includes various cases where `role="alert"`, `role="status"`, etc. may be needed.

In almost all of the usages of this component that I can imagine from products we've seen on VA.gov, no `role` attribute will be required. "Static alert: no role" is the first bullet point in this section of the guidance, but I'm worried that it'll be lost next to the longer descriptions of when various roles would make sense.

## Recommended action
Consider organizing this section to de-emphasize the other `role` situations and emphasize that it will usually not be required. Would it be possible to put the other role situations in additional info components?

## References

- **Accessibility Defect Severity:** N/A
- **WCAG Success Criteria:** N/A
- **Modality:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A

---

### Next Steps for DST

- Close the ticket when the issue has been resolved or validated by your Product Owner
- If your team has additional questions or needs Governance help validating the issue, please comment on the ticket
- If this ticket has a `high-priority` label, please address as soon as possible so VFS teams are not impacted
- If this ticket has a `consider` label, please consider for future implementation
